### PR TITLE
AWS Provider 

### PR DESCRIPTION
**Pull request by Tiboris**
Add AWS Provider 
Slightly refactor imports

### CODE REVIEW

File: src/aiohabit/transformers/__init__.py [hunk 1]
```diff
@@ -56,3 +60,4 @@ def names(self):
 
 transformers = Registry()
 transformers.register(OPENSTACK_KEY, OpenStackTransformer)
+transformers.register(AWS_KEY, AWSTransformer)
```
[the remark this answers is not in the document]
Comment (Tiboris) [reply src/aiohabit/transformers/__init__.py:63]: I am also more leaned to init that in __init__.py i believe we can later preprocess and init only needed provisioners I will move it there and refactor some things related to your comments.

File: src/aiohabit/providers/aws.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+        self.poll_sleep = 7  # seconds
+
+    @property
+    def name(self):
+        """Get provider name."""
+        return self._name
+
+    async def init(self, image_names):
+        """Initialize provider with data from AWS."""
+        # AWS_CONFIG_FILE=`readlink -f ./aws.key`
```
[the remark this answers is not in the document]
Comment (Tiboris) [reply src/aiohabit/providers/aws.py:71]: OpenStack provider as you said loads the settings form env vars which needs to be sourced before run (https://github.com/pvoborni/aiohabit/blob/66b40400bd2aa87105666362e77caa9a75ff186e/src/aiohabit/providers/openstack.py#L87)

File: src/aiohabit/providers/aws.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            Tags=[
+                {"Key": "Name", "Value": "IDM-CI-runner"},
+                {"Key": "name", "Value": specs.get("name")},
+                {"Key": "IDM-CI", "Value": "True"},
+            ],
+        )
+        # returns id of provisioned instance
+        return ids[0]
+
+    def host_info_from_prov_result(self, prov_result):
```
[the remark this answers is not in the document]
Comment (Tiboris) [reply src/aiohabit/providers/aws.py:114]: No problem!

File: src/aiohabit/providers/aws.py [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+            "id": None,
+            "name": None,
+            "addresses": None,
+            "status": None,
+            "fault": None,
+        }
+        result["id"] = prov_result.get("InstanceId")
+        result["name"] = [
+            i["Value"] for i in prov_result.get("Tags") if i["Key"] == "name"
+        ].pop()  # should be one key "name"
```
[the remark this answers is not in the document]
Comment (Tiboris) [reply src/aiohabit/providers/aws.py:127]: Agreed

File: tests/unittests/test_openstack.py [hunk 5]
```diff
@@ -15,7 +15,7 @@
 from aiohabit.providers.openstack import OpenStackProvider
 from unittest import mock
 from unittest.mock import Mock, patch
-from .utils import get_data
+from aiohabit.utils import get_data
```
Comment (netoarmando) [inline tests/unittests/test_openstack.py:18]:
```python
from unittests.utils import get_data
```

Comment (Tiboris) [reply tests/unittests/test_openstack.py]: This needs to be fixed

**Pull request merged**